solx/src/solx/arguments.rs

```diff
@@ -203,10 +203,34 @@ impl Arguments {
     pub fn validate(&self) -> Arc<Mutex<Vec<solx_standard_json::OutputError>>> {
         let mut messages = vec![];
 
-        if self.version && std::env::args().count() > 2 {
+        if self.recursive_process {
+            if std::env::args().count() > 3 {
+                messages.push(solx_standard_json::OutputError::new_error(
```

Comment:
It looks like `solx_standard_json::OutputError::new_error` is frequently called with `None` as the 1st, 3rd, and 4th parameter. Making a wrapper for it would reduce duplication.